plugins/node/opentelemetry-instrumentation-cassandra/test/cassandra-driver.test.ts

```diff
@@ -76,6 +86,18 @@ function assertSingleSpan(name: string, query?: string, status?: SpanStatus) {
   assertSpan(span, name, query, status);
 }
 
+function assertAttributeInSpan(
```

Comment:
This function also asserts that there is only 1 span, but the name does not explain this.

In the current implementation, it asserts more than attributes (span name, query, status). Can be nice to rename this function to something that better describes what it does